cookbooks/aws-parallelcluster-environment/attributes/environment.rb

```diff
@@ -61,4 +61,7 @@
 
 default['cluster']['head_node_private_ip'] = nil
 
+# TODO: Move to platform cookbook
 default['cluster']['spack_shared_dir'] = "#{node['cluster']['shared_dir']}/spack"
```

Comment:
We should align to the same attribute naming convention. Will address in a dedicated PR